cypress/full/buttons_page_spec.ts

```diff
@@ -0,0 +1,213 @@
+      buttons.isBtnCheckboxSelected(checkboxDemo, 2, false);
+    });
+  });
+
+  describe('Custom checkbox value', () => {
+    const customCheckboxVal = buttons.exampleDemosArr.customCheckboxVal;
+
+    it('example contains btn and output window', () => {
+      buttons.isButtonVisible(customCheckboxVal, 'Single Toggle');
+      buttons.isPreviewExist(customCheckboxVal, '1');
+    });
+
+    it('output value , can be changed by click on button', () => {
+      buttons.isButtonEnabled(customCheckboxVal, 'Single Toggle');
+      buttons.isPreviewExist(customCheckboxVal, '1');
+      buttons.clickOnBtn(customCheckboxVal);
+      buttons.isPreviewExist(customCheckboxVal, '0');
+    });
+  });
+
+  describe('Checkbox with Reactive Form', () => {
+    const checkboxWithForm = buttons.exampleDemosArr.checkboxWithForms;
+
+    it('example contains output window and btn group', () => {
+      buttons.isPreviewExist(checkboxWithForm, '"left": false,\n' +
+        '  "middle": true,\n' +
+        '  "right": false');
+      buttons.isButtonsVisibleInGroup(checkboxWithForm, 'Left');
+      buttons.isButtonsVisibleInGroup(checkboxWithForm, 'Right');
+      buttons.isButtonsVisibleInGroup(checkboxWithForm, 'Middle');
+    });
+
+    it('when user clicks on the Left button it became active and after user clicks again - inactive', () => {
+      buttons.isButtonsVisibleInGroup(checkboxWithForm, 'Left');
+      buttons.isBtnCheckboxSelected(checkboxWithForm, 0, false);
+      buttons.clickByText(checkboxWithForm, 'Left');
+      buttons.isBtnCheckboxSelected(checkboxWithForm, 0, true);
+      buttons.clickByText(checkboxWithForm, 'Left');
+      buttons.isBtnCheckboxSelected(checkboxWithForm, 0, false);
+    });
+
+    it('when user clicks on the Middle button it becomes inactive and all buttons are inactive', () => {
+      buttons.isButtonsVisibleInGroup(checkboxWithForm, 'Middle');
+      buttons.isBtnCheckboxSelected(checkboxWithForm, 1, true);
+      buttons.clickByText(checkboxWithForm, 'Middle');
+      buttons.isBtnCheckboxSelected(checkboxWithForm, 1, false);
+      buttons.isBtnCheckboxSelected(checkboxWithForm, 0, false);
+      buttons.isBtnCheckboxSelected(checkboxWithForm, 2, false);
+    });
+
+    it('when user clicks on the Right button it became active and after user clicks again - inactive', () => {
+      buttons.isButtonsVisibleInGroup(checkboxWithForm, 'Right');
+      buttons.isBtnCheckboxSelected(checkboxWithForm, 2, false);
+      buttons.clickByText(checkboxWithForm, 'Right');
+      buttons.isBtnCheckboxSelected(checkboxWithForm, 2, true);
+      buttons.clickByText(checkboxWithForm, 'Right');
+      buttons.isBtnCheckboxSelected(checkboxWithForm, 2, false);
+    });
+  });
+
+  describe('Radio buttons', () => {
+    const radioCheck = buttons.exampleDemosArr.radioBtn;
+    const btnRadioGroup = buttons.btnRadioGroup;
+
+    it('example contains btns, Radio btns and checkbox', () => {
```

Comment:
example contains 2 button-groups (primary and success), contain of 3 parts(labels) and 1 preview